conans/client/package_layouts/package_user_layout.py

```diff
@@ -0,0 +1,17 @@
+# coding=utf-8
+
+from conans.util.files import load
```

Comment:
instead of making a package ``package_layouts``, I would put the layouts inside the new ``paths`` package.